libs/multiplayer/mp.ts

```diff
@@ -1,3 +1,4 @@
+const postPresence = multiplayer.postPresenceIcon;
```

Comment:
This is a hack to get around the `multiplayer` namespace getting shadowed by `sprites.multiplayer`, won't merge in as is~